### ISSUE TITLE
ReactDOM.createPortal instead of ReactDOM.render

### ISSUE THREAD

**Issue by osi-oswald** [akxcv/vuera#17]
Did you already try the new [ReactDOM.createPortal](https://reactjs.org/docs/portals.html) as replacement for ReactDOM.render? 

The benefits would be to have only 1 react app for all ReactInVue components (instead of N reacts apps, one for each ReactInVue component). Also one would be able to define a single react context to be used by all ReactInVue components. Maybe this will have some (positive) impact on your planned Benchmark #16?

**Comment by akxcv** [comment]
Yeah, I’ve been thinking about that for a long time. I haven’t thought about preserving context this way, though, thanks for the idea! I believe we can achieve similar effect for Vue components via Vue slots. But I would still like to benchmark the library first.

**Comment by thany** [comment]
It's a good idea to revive this issue becuase of that error that now appears in the browser console:

```
Warning: ReactDOM.render is no longer supported in React 18. Use createRoot instead. Until you switch to the new API, your app will behave as if it's running React 17. Learn more: https://reactjs.org/link/switch-to-createroot
```

**Comment by another-strange-cat** [comment]
Hi, I'd like to work on this.

**Comment by GitsOfVivek** [comment]
is it still open?

**Comment by cwadrupldijjit** [comment]
In theory, yes.  I might need to revisit this a bit.  I might include it in my updates to this package.
